it/auth/SelfAssessmentIndIndividualsDetailsApiAuthMainAgentsOnlyISpec.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIED OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package auth
+
+import play.api.libs.json.{JsValue, Json}
+import play.api.libs.ws.{WSRequest, WSResponse}
+import shared.auth.AuthMainAgentsOnlyISpec
+import shared.services.DownstreamStub
+
+class SelfAssessmentIndIndividualsDetailsApiAuthMainAgentsOnlyISpec extends AuthMainAgentsOnlyISpec {
```

Comment:
Typo in both class/file name, needs to be SelfAssessmentIndividualDetailsApiAuthMainAgentsOnlyISpec